test/language/class/class-default-constructor-no-crash.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
```

Comment:
I'm not sure this is interesting from a test262 perspective. Would be ok removing this test.